### DISCUSSION TITLE
Star ratings for all graphs in Artsdata

### DISCUSSION THREAD

**Discussion by saumier** [culturecreates/artsdata-data-model#26] [Ideas]
I'm thinking the time is right to add a star rating to graphs in Artsdata. The goal is to inform data consumers about the quality of the data from a website or database source. For example, a data consumer like LaVitrine that wants to make a calendar of events in Quebec, could decide to include all 4 star graphs from Quebec in their data feed.  

My question to all of you is what criteria  should be used to determine the star rating. 

Can you propose things that you think a data consumer would need to know about the quality of data to make a decision to use it?

**Comment by troughc** [comment]
I agree, the timing is right. As for what increases the start rating - def starting point is having all the required metadata for google events. Other thoughts are appropriate length descriptions; 'good' image; and whatever the consumer is looking for, i.e. we need to know what their needs/expectations are for their sites and then determine a star rating based on how well the graph meets those needs. cc @saumier

**Reply by saumier** [reply]
Thanks. I agree that Events should have the required metadata as described by Google Events. That could earn 1 star. Another start could be for all recommended properties...

**Reply by tammy-culture** [reply]
Is this an example of what you are talking about?

1 star: Title, date, location

2 star: 1 star plus event type

3 star: 2 star plus name of performing artist or company

4 star: 3 star plus event description

5 star: 4 star plus promotional images

- Title
- Date
- Location
- Event type
- Name of performing artist or company
- Event description
- Promotional image

**Reply by tammy-culture** [reply]
What if we offered something like the following?  Where there are 3 levels: 
(1) Basic : Title, date, location, event type
(2) Basic plus : Title, date, location, event type, name of performing artist or company
(3) Premium : Title, date, location, event type, name of performing artist or company, event description, promotional image

**Comment by saumier** [comment]
I propose we create a spreadsheet and list the attributes we want.  We can indicate if a property is present or not, but there could also be a notion of quality. For example, an image URL may be present but be a broken link. If we could say the image is present and works then that is a measure of the quality. Another simple case is when attendanceMode is present but wrong, and whether the graph is reviewed by a human or not. Also the presence of a CC0 license is important to list.
